dask_cuda/tests/test_device_host_file.py

```diff
@@ -154,9 +154,7 @@ def test_serialize_cupy_collection(collection, length, value):
     else:
         obj = device_to_host(collection((x,) * length))
 
-    if length > 5:
-        assert obj.header["serializer"] == "pickle"
-    elif length > 0:
+    if length > 0:
```

Comment:
can you remind me what's going on here ?  did we use to serialize with pickle for small objects ?

Reply:
Yes, but this magic number was now fixed in https://github.com/dask/distributed/pull/3689 , so everything is serialized using the default `"dask"` serializer.

Reply:
Thanks!